prow/create-build-cluster.sh

```diff
@@ -54,6 +54,19 @@ ADMIN_IAM_MEMBER="${ADMIN_IAM_MEMBER:-group:mdb.cloud-kubernetes-engprod-oncall@
 # Overriding output
 OUT_FILE="${OUT_FILE:-build-cluster-kubeconfig.yaml}"
 
+
+# Require bash version >= 4.4
+if [[ -z "${BASH_VERSINFO[@]}" ]]; then
+  echo "FAILURE: Couldn't determine bash version from \$BASH_VERSINFO. Please install bash version >= 4.4"
+  exit 1
+else
+  ver="${BASH_VERSINFO[0]}.${BASH_VERSINFO[1]}"
+  if (( $(echo "${ver} < 4.4" | bc -l) )); then
```

Comment:
we actually already have this implemented in kubernetes/kubernetes (though for 4.2)
https://github.com/kubernetes/kubernetes/blob/ea0764452222146c47ec826977f49d7001b0ea8c/hack/lib/util.sh#L725-L734

that implementation is pretty good, and I think the prompt to install with homebrew is helpful, FWIW.

Reply:
Agreed, I used that instead.